netci.groovy

```diff
@@ -2067,7 +2069,9 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                         // Build and text corefx
                         def workspaceRelativeFxRoot = "_/fx"
                         def absoluteFxRoot = "\$WORKSPACE/${workspaceRelativeFxRoot}"
-                        buildCommands += "python \$WORKSPACE/tests/scripts/run-corefx-tests.py -arch ${arch} -build_type ${configuration} -fx_root ${absoluteFxRoot} -fx_branch ${branch} -env_script ${scriptFileName}"
+                        def coreFxSpecificCommit = (branch == 'master') ? '551fe49174378adcbf785c0ab12fc69355cef6e8' : 'HEAD'
```

Comment:
I would prefer to hard code this in the python since it's so hard to test groovy changes.  No need to plumb this through the script command line.  If you're rushed, you can go ahead, though.  Sorry for the delay reviewing this.